duet/sys/config-user.g

```diff
@@ -0,0 +1,67 @@
+;File     : config-user.g
```

Comment:
After talking with @lostapathy, I think we should not include this file at all. Maybe provide a Project R3d focused example and allow self-builders to construct their own.

Maybe even better, include only the following gcode in this file:
```
M291 P"You do not have a user configuration! This must be created before proceeding" R"Warning!" S2 ; Message waits for response before continuing
M112 ; Emergency stop so they don't hurt themselves
```

or something similar.

Reply:
The user experience would look like this when browsing to DWC (Since we cannot assume PanelDue):

<img width="1293" alt="Screenshot 2019-05-05 11 19 30" src="https://user-images.githubusercontent.com/867868/57196129-d0dc0a80-6f27-11e9-803b-c1cc26c7363b.png">
<img width="1293" alt="Screenshot 2019-05-05 11 19 37" src="https://user-images.githubusercontent.com/867868/57196130-d33e6480-6f27-11e9-8d67-03e74d239ef9.png">

Reply:
Basically this puts them into a boot loop until they remove the SD, read the README.md and replace the `config-user.g` file

Reply:
Not sure I like putting them in a boot loop that requires SD card removal - getting that thing out of the electronics cover is a chore.

Reply:
I thought about that, but if the choice is between safety and minor annoyance, I'd rather err to safety.

Also, we could note in the README or elsewhere they can press `esc` to bypass the warning rather than removing the SD.

Reply:
My take is to provide consructed example files , one for the kit and one for self builders (such as my original, maybe with more comments)?
It'd speed up builds if people can think "I have a ZLT with an E3D Gold hotend, a Piezo probe, a TR8*2 leadscrew with 0.9 deg steppers and a BMG extruder,  I can just copy those code blocks from the example file" and they are 99% there.